content/doc/developer/testing/index.adoc

```diff
@@ -246,6 +246,48 @@ To recreate the same configuration from a test method, you can do the following:
 In order to test parts of your plugin, you may want certain files to exist in the build workspace, or that Jenkins is configured in a certain way.
 This section covers various ways to achieve this using the Jenkins Test Harness.
 
+=== Operating System Awareness
+
+Sometimes it ios necessary to adjust the testcase depending on the operating system.
+This can be the choice of Batch or Shell commands.
```

Comment:
More words to note that there are other cases that are specific to one operating system or another.
```suggestion
This can be the choice of Batch or Shell commands or other test conditions that are operating system specific.
```